packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -411,6 +413,36 @@ class ChromiumLauncher {
     return chrome;
   }
 
+  /// Gets the first [chrome] tab.
+  ///
+  /// Note: Retry getting tabs from Chrome for a few seconds and retry finding
+  /// the tab a few times. This prevents flakes caused by Chrome not returning
+  /// correct output if the call was too close to the start.
+  Future<ChromeTab?> _getFirstTab(Chromium chrome) async {
+    const Duration retryFor = Duration(seconds: 2);
+    const int attempts = 5;
+
+    for (int i = 1; i <= attempts; i++) {
+      try {
+        final List<ChromeTab> tabs =
+          await chrome.chromeConnection.getTabs(retryFor: retryFor);
+
+        if (tabs.isNotEmpty) {
+          return tabs.first;
+        }
+        if (i == attempts) {
+          return null;
+        }
+      } on ConnectionException catch (_) {
+        if (i == attempts) {
+          rethrow;
+        }
+      }
+      await Future<dynamic>.delayed(const Duration(milliseconds: 25));
```

Comment:
```suggestion
      await Future<void>.delayed(const Duration(milliseconds: 25));
```

Reply:
we try to avoid `dynamic` in the tool (in cases where you actually use hte value, `Object?` is preferred over `dynamic`)